### PR TITLE
 fix(Inputmask):fixes defaultValue prop #250

### PR DESCRIPTION
**Pull request by stutirao**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/molecules/inputMask/InputMask.tsx [hunk 1]
```diff
@@ -28,12 +28,12 @@ export interface MaskProps extends BaseProps {
   onClear?: (e: React.MouseEvent<HTMLElement>) => void;
 }
 
-export type InputMaskProps = InputProps & MaskProps;
+export type InputMaskProps = Omit<InputProps, 'value'> & MaskProps;
```
Comment (riyalohia) [inline core/components/molecules/inputMask/InputMask.tsx]: Add a description.

File: core/components/molecules/inputMask/InputMask.tsx [hunk 2]
```diff
@@ -28,12 +28,12 @@ export interface MaskProps extends BaseProps {
...
 
 export const InputMask = React.forwardRef<HTMLInputElement, InputMaskProps>((props, forwardRef) => {
   const {
     mask: maskProp,
-    value: valueProp,
+    defaultValue: defaultValue,
```
Comment (riyalohia) [inline core/components/molecules/inputMask/InputMask.tsx]: Replace `defaultValue: defaultValue` with defaultValue

File: core/components/molecules/inputMask/InputMask.tsx [hunk 3]
```diff
@@ -26,14 +26,17 @@ export interface MaskProps extends BaseProps {
...
   onBlur?: (e: React.ChangeEvent<HTMLInputElement>, val?: string) => void;
   onClear?: (e: React.MouseEvent<HTMLElement>) => void;
-}
 
-export type InputMaskProps = InputProps & MaskProps;
+}
 
+export type InputMaskProps = Omit<InputProps, 'value'> & MaskProps;
+/**
+ * inputMask works as uncontrolled component
```
Comment (aditya-kumawat) [inline core/components/molecules/inputMask/InputMask.tsx]: InputMask*

**Comment by aditya-kumawat** [comment]
Use following command for commit:
git commit -m "[COMMIT_TYPE](COMMIT_SCOPE): [COMMIT_DESC]" -m "Mentions/Closes ISSUE_ID"

**Pull request closed without merging**